src/test/objectstore/Allocator_bench.cc

```diff
@@ -16,6 +16,7 @@
 
 #include <boost/random/mersenne_twister.hpp>
 #include <boost/random/uniform_int.hpp>
+#include <boost/random/mersenne_twister.hpp>
```

Comment:
Isn't this header already included two lines above?  The same goes for the other two hunks.

Reply:
Thank you, looks like someone else got to it, first.